sys/net/gnrc/routing/rpl/gnrc_rpl_dodag.c

```diff
@@ -66,6 +66,15 @@ static void _rpl_trickle_send_dio(void *args)
           ipv6_addr_to_str(addr_str,&dodag->dodag_id, sizeof(addr_str)));
 }
 
+/* The lifetime of the default route should exceed the parent timeout interval
+ * by the time we allow the node to probe its parent */
+static uint16_t _dflt_route_lifetime(gnrc_rpl_dodag_t *dodag)
```

Comment:
What unit is the return type of this function?
maybe use `_dflt_route_lifetime_ms()`

Reply:
can do.